whipper/common/program.py

```diff
@@ -215,59 +215,38 @@ def getPath(self, outdir, template, mbdiscid, i, disambiguate=False):
         v['x'] = 'flac'
         v['X'] = v['x'].upper()
         v['y'] = '0000'
+        if track_number:
```

Comment:
Re the HTOA bug: when `track_number == 0`, this will evaluate to `False`.  Thus line 222 is dead code. Maybe change to `if track_number >= 0:` ?

Reply:
Thank you!